bpf/include/bpf/config/node.h

```diff
@@ -29,3 +29,5 @@ NODE_CONFIG(__u32, direct_routing_dev_ifindex, "Index of the interface used to c
 
 NODE_CONFIG(bool, supports_fib_lookup_skip_neigh,
 	    "Whether or not BPF_FIB_LOOKUP_SKIP_NEIGH is supported.")
+NODE_CONFIG(__u32, wg_ifindex, "Index of the WireGuard interface.")
+NODE_CONFIG(__u16, wg_port, "Port for the WireGuard interface.")
```

Comment:
This is wireguard specific config only used by the logic in `bpf/lib/wireguard.h`, so i would suggest making these into `DECLARE_CONFIG` instead of `NODE_CONFIG` and moving that declaration to `bpf/lib/wireguard.h`.

The generation process then adds the config to only the program configs that include `bpf/lib/wireguard.h` such as `bpf_host.c` / `BPFHost`

Reply:
Done